R/cuda.R

```diff
@@ -117,6 +117,13 @@ dust_cuda_configuration <- function(path_cuda_lib = NULL,
 ##'   docs](https://docs.nvidia.com/cuda/cuda-compiler-driver-nvcc/index.html)
 ##'   for more details.
 ##'
+##' @param flags Optional extra arguments to pass to nvcc. These
+##'   options will not be passed to your normal C++ compiler, nor the
+##'   linker (for that use R's user Makevars system). This can be used
+##'   to do things like tune the maximum number of registers
+##'   (`--maxregcount=x`). If not `NULL`, this must be a character
```

Comment:
```suggestion
##'   (`--maxrregcount x`). If not `NULL`, this must be a character
```

(I don't know why...)